src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -0,0 +1,56 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {setWorkspaceRequiresCategory} from '@libs/actions/Policy';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceCategoriesSettingsPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES_SETTINGS>;
+
+function WorkspaceCategoriesSettingsPage({route}: WorkspaceCategoriesSettingsPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const updateWorkspaceRequiresCategory = (value: boolean) => {
+        setWorkspaceRequiresCategory(route.params.policyID, value);
+    };
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+                {({policy}) => (
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        style={[styles.defaultModalContainer]}
+                        testID={WorkspaceCategoriesSettingsPage.displayName}
+                        shouldShowOfflineIndicatorInWideScreen
```

Comment:
@ArekChr I'm not sure that the last commit was correct. I can still see the offline message. I think we should revert that so we still have the `OfflineWithFeedback` component to apply the offline/optimistic styles to the switch and instead just remove this line
```suggestion
```

Reply:
You are right, the name of the component confused me, fixing!